JavaErickHernandez/Day3/RansomNote.java

```diff
@@ -0,0 +1,88 @@
+package bootcamp;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Scanner;
+
+public class RansomNote {
+	final static String wordsMagazine = "Enter the number of words in the magazine: ";
+	final static String wordsNote = "Enter the number of words in the note: ";
+	final static String word = "Enter the word: ";
+	final static String yes = "YES";
+	final static String no = "NO";
+	final static String tryAgainMessage = "Try Again";
+	final static HashSet<String> noteWordSet = new HashSet<String>();
```

Comment:
Always using interface type when declaring a collection

Reply:
I had read the info you share, here is the example 
![image](https://user-images.githubusercontent.com/49655011/131871164-32f74fa6-b333-4f41-973d-d6512ce26123.png)
Im not sure what should I change... I guess like.... : 
final static HashSet<String> noteWordSet = new ArrayList<String>();

Reply:
c'mon!!! what is the interface??? Is HashSet an interface?